session.go

```diff
@@ -257,6 +277,8 @@ func (s *Session) NewSender(ctx context.Context, target string, opts *SenderOpti
 		return nil, err
 	}
 
+	go l.mux(senderTestHooks{})
```

Comment:
I meant to ask - was this just a bug before? Where did the mux get launched previously?

Reply:
Yeah, I noticed it was "symmetrical" :) Awesome.